src/ServiceManagement/Services/Commands.Utilities/Websites/Services/WebEntities/SiteConfig.cs

```diff
@@ -63,7 +63,7 @@ public interface ISiteConfig
 
         bool? RemoteDebuggingEnabled { get; set; }
 
-        RemoteDebuggingVersion RemoteDebuggingVersion { get; set; }
```

Comment:
Is the type "RemoteDebugginVersion" used anywhere stil? If not we should delete the type.